frame/elections-phragmen/src/lib.rs

```diff
@@ -682,6 +682,10 @@ decl_event!(
 		/// A \[member\] has been removed. This should always be followed by either `NewTerm` or
 		/// `EmptyTerm`.
 		MemberKicked(AccountId),
+		/// A candidate was slashed due to failing to obtain a seat as member or runner-up
+		CandidateSlashed(AccountId, Balance),
+		/// A member/runner-up was slashed due to failing to retaining their position.
+		MemberSlashed(AccountId, Balance),
```

Comment:
```suggestion
		SeatHolderSlashed(AccountId, Balance),
```

Maybe this @thiolliere?